src/pages/App/App.js

```diff
@@ -10,13 +10,12 @@ import ScreenshotModal from '../../components/ScreenshotModal/ScreenshotModal';
 import Error from '../../components/Error/Error';
 import Success from '../../components/Success/Success';
 import ShowElementsOnFullSize from '../../utils/ShowElementsOnFullSize';
+import { setCurrentTime, setDuration, setPlayPressed } from '../../store/actions/songActions';
 
-export default function App({ song }) {
+export default function App() {
     // Local States
-    const [playPressed, setPlayPressed] = useState(false);
+    const [uploadedSong, setUploadedSong] = useState(null);
```

Comment:
This piece of state has been deleted because it is just a duplication of redux song state. Pls don't re-add it

Reply:
done!